src/map/battle.cpp

```diff
@@ -8582,7 +8582,7 @@ void battle_adjust_conf()
 
 #if PACKETVER > 20120000 && PACKETVER < 20130515 /* Exact date (when it started) not known */
 	if (battle_config.feature_auction) {
-		ShowWarning("conf/battle/feature.conf:feature.auction is enabled but it is not stable on PACKETVER "EXPAND_AND_QUOTE(PACKETVER)", disabling...\n");
```

Comment:
Just add a space between the quotation mark and the macro. Making it an argument is not necessary.

Reply:
lol didn't knew that the case XD